package com.esgi.nova.core_api.user.exceptions

class UserNotFoundByUsernameException : Throwable() {

}